lib/timeutils/tests/test_unixtime.c

```diff
@@ -60,12 +61,12 @@ Test(unixtime, unix_time_fix_timezone_adjusts_timestamp_as_if_was_parsed_assumin
 
   _wct_initialize(&wct, "Jan 19 2019 18:58:48");
   wct.wct_gmtoff = 3600;
-  convert_wall_clock_time_to_unix_time(&ut, &wct);
+  convert_wall_clock_time_to_unix_time(&wct, &ut);
```

Comment:
As I see the variable orders are fixups for the previous commit: `unixtime: add unix_time_{set,fix}_timezone() functions and associated testing`.

Reply:
that's right, lost during the rebase. This is fixed now.